build.gradle

```diff
@@ -87,13 +87,20 @@ subprojects { project ->
   }
 }
 
+task deployMeta {
+  dependsOn ':compiler-plugin:publishToMavenLocal'
+  dependsOn ':gradle-plugin:publishArrowPluginMarkerMavenPublicationToMavenLocal'
+  dependsOn ':gradle-plugin:publishPluginMavenPublicationToMavenLocal'
+  dependsOn ':gradle-plugin:publishToMavenLocal'
+}
```

Comment:
It seems this task could be `installMeta` because it's publishing to the local repository.